src/style.css

```diff
@@ -1,3 +1,63 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  text-transform: uppercase;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: white;
+  padding: 0 50px;
+  width: 100% - 100px;
```

Comment:
```suggestion
```
why do you need such styles for widht?  header width will be on full width of body as it is block element.